xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/java/org/xwiki/livedata/internal/livetable/LiveTableLiveDataEntryStore.java

```diff
@@ -190,4 +199,110 @@ private String getLiveTableResultsFromPage(String page, LiveDataQuery query) thr
+            boolean hasChanged = false;
+            for (Map.Entry<String, Object> newProperty : newProperties.entrySet()) {
+                String propertyKey = newProperty.getKey();
+                // Update only if the entry exists in the object
+                if (properties.contains(propertyKey)) {
+                    try {
+                        firstObject.set(propertyKey, newProperty.getValue(), xcontext);
+                        hasChanged = true;
+                    } catch (java.lang.ClassCastException e) {
+                        // TODO: handle the case where wrong values are set for some properties.
+
+                    }
+                }
+            }
+            if (hasChanged) {
+                xcontext.getWiki().saveDocument(document, xcontext);
+            }
+        } catch (AccessDeniedException | XWikiException e) {
+            // TODO log
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public Optional<LiveDataEntryStoreEditDescriptor> getEdit(String entryId, String propertyId)
+    {
+        String className = (String) ((List) this.getParameters().get(CLASS_NAME_PARAMETER)).get(0);
+        DocumentReference documentReference = this.currentDocumentReferenceResolver.resolve(entryId);
+        DocumentReference classReference = this.currentDocumentReferenceResolver.resolve(className);
+        try {
+            this.authorization.checkAccess(Right.VIEW, documentReference);
+            XWikiContext xcontext = this.xcontextProvider.get();
+            XWikiDocument document = xcontext.getWiki().getDocument(documentReference, xcontext);
+            BaseObject object = document.getXObject(classReference);
+
+            XWikiDocument oldDoc = xcontext.getDoc();
+            try {
+                xcontext.setDoc(document);
+                String body = object.displayEdit(propertyId, xcontext);
+
+                // TODO: probably not the simplest solution
+                String dependencies = loadDependencies(xcontext);
+
+                LiveDataEntryStoreEditDescriptor liveDataEntryStoreEditDescriptor = new LiveDataEntryStoreEditDescriptor();
+                liveDataEntryStoreEditDescriptor.setBody(body);
+                liveDataEntryStoreEditDescriptor.setDependencies(dependencies);
+                return Optional.of(liveDataEntryStoreEditDescriptor);
+            } finally {
+                xcontext.setDoc(oldDoc);
+            }
+        } catch (AccessDeniedException | XWikiException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private String loadDependencies(XWikiContext xcontext)
+    {
+        String importString = "";
+        for (String s : Arrays.asList("ssrx", "ssfx", "ssx", "linkx", "jsrx", "jsfx", "jsx")) {
+
+            importString +=
+                ((SkinExtensionPluginApi) xcontext.getWiki().getPluginApi(s, xcontext)).getImportString();
+        }
+        return importString;
+    }
+
```

Comment:
Iintialize the javascript and css dependencies of the edit form. This is not working well, and the dependencies are always returned even if they have already been returned for a previous request on the same page.